external_plugins/track_and_trace/track_and_trace_plugin.cpp

```diff
@@ -122,9 +122,9 @@ void track_and_trace_plugin::plugin_initialize( const boost::program_options::va
    try
    {
       ilog( "Initializing track_and_trace_plugin_impl plugin" );
-      chain::database& db = appbase::app().get_plugin< sophiatx::plugins::chain::chain_plugin >().db();
+      auto db = appbase::app().get_plugin< sophiatx::plugins::chain::chain_plugin >().db();
```

Comment:
db here is going to be shared_ptr<...> according to deduction rules. auto& might better so there is no copy of shared_ptr made.